app/src/main/java/com/android/solvit/provider/ui/request/RequestsFeed.kt

```diff
@@ -342,6 +534,7 @@ fun ServiceRequestItem(
  * @param showDialog The showDialog state
  * @param requestViewModel The service request view model
  */
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun ProposePackageDialog(
```

Comment:
I feel like this makes the code not readable maybe exctract the logic smaller composables like:
PackageList()
PackageSelectionButton()
PriceInputField()

Reply:
You are right, it is a valid concern. I will keep that in mind so I can implement a more compartmented version of this composable in the next sprint.